src/portals/hope/features/claims/claims-list/LargeClaimsList.tsx

```diff
@@ -110,6 +114,28 @@ export const LargeClaimsList: React.FC<{
     })
   }, [page, date, settings, filters, loadingPermission])
 
+  const search = () => {
+    if (loadingPermission) {
+      return
+    }
+    const settingsFilters = {
+      filterCreatedBeforeOrOnDate: date,
+      filterClaimStates: settings.claimStatesFilterClaims,
+      filterComplexities: settings.claimComplexityFilterClaims,
+      filterMarkets: settings.marketFilterClaims,
+      filterTypesOfContract: hasCarPermission
+        ? TypeOfContractByInsuranceType.SWEDISH_CAR
+        : null,
+      filterClaimOutcomes: settings.outcomeFilterClaims,
+      searchString,
+    }
+
+    listClaims({
+      page: page - 1 ?? 0,
```

Comment:
How would `page - 1` be `undefined` or `null`?  `undefined - 1` is `NaN`
```suggestion
      page: page - 1 ,
```

Reply:
Does `??` work on `NaN`?

Reply:
Good point. This was a lazy copy of the existing implementation. I'll correct it